packages/flutter_test/lib/src/_goldens_io.dart

```diff
@@ -0,0 +1,209 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:async';
+import 'dart:io';
+import 'dart:math' as math;
+import 'dart:typed_data';
+
+import 'package:image/image.dart';
+import 'package:meta/meta.dart';
+import 'package:path/path.dart' as path;
+import 'package:test_api/test_api.dart' as test_package show TestFailure;
+
+import 'goldens.dart';
+
+/// The default [GoldenFileComparator] implementation for `flutter test`.
```

Comment:
There are doc updates that landed in https://github.com/flutter/flutter/pull/39082 for `GoldenFileComparator` and `LocalFileComparator`.